packages/augur-ui/src/modules/market-cards/sports-market-card.tsx

```diff
@@ -41,11 +41,12 @@ const { MONEY_LINE } = SPORTS_GROUP_MARKET_TYPES;
 
 const determineStartState = ({ type, marketTypes }) => {
   if (type === FUTURES) return true;
-  if (type !== DAILY) return false;
-  const hasMoneyLineMarket = marketTypes.find(uniqueType =>
-    uniqueType.includes(MONEY_LINE)
-  );
-  return !hasMoneyLineMarket;
+  // if (type !== DAILY) return false;
+  // const hasMoneyLineMarket = marketTypes.find(uniqueType =>
+  //   uniqueType.includes(MONEY_LINE)
+  // );
```

Comment:
Is it supposed to be commented out?

Reply:
yeah, I might need to add that logic back in, i'll remove it in another pr after i'm sure I don't